build/postgres/Dockerfile

```diff
@@ -51,6 +91,8 @@ else \
 		psmisc \
 		rsync \
 		wal2json${PG_MAJOR//.} \
+		file \
+		unzip \
```

Comment:
This block also needs a `pgaudit_analyze`

Reply:
for the "non-centos" block pgaudit_analyze is added as part of a third install block on this [line](https://github.com/CrunchyData/crunchy-containers/blob/0f4bc8090e4ee30a2d602e4a2f531daad0357c88/build/postgres/Dockerfile#L57). The only difference between the second and third block is the third has a `--setopt=tsflags=''`.